src/odemis/util/conversion.py

```diff
@@ -502,17 +502,17 @@ def represent_numpy_floating(self, data):
 class YamlExtraDumper(Emitter, Serializer, CustomSafeRepresenter, Resolver):
 
     def __init__(self, stream,
-            default_style=None, default_flow_style=False,
-            canonical=None, indent=None, width=None,
-            allow_unicode=None, line_break=None,
-            encoding=None, explicit_start=None, explicit_end=None,
-            version=None, tags=None):
+                 default_style=None, default_flow_style=False,
+                 canonical=None, indent=None, width=None,
+                 allow_unicode=None, line_break=None,
+                 encoding=None, explicit_start=None, explicit_end=None,
+                 version=None, tags=None, **kwargs):
```

Comment:
Let's add a comment that it's supposed to catch "sort_keys" on newer versions of PyYaml.